packages/core/package.json

```diff
@@ -16,19 +16,13 @@
     "http-status-codes": "2.2.0",
     "idb": "7.1.0",
     "logdown": "3.3.1",
-    "long": "4.0.0",
```

Comment:
I do believe we use `long` in the core, why did you remove it? 🤔

Reply:
i added it back in